### PR TITLE
chore(typescript): setup to enable typescript

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export interface Translator {
+  get: (str: string, args?: string[]) => string;
```
Comment (haxxmaxx) [inline src/types.ts:2]: this should be a type in nebula?

Comment (LiKang6688) [reply src/types.ts:2]: unfortunately, no. nebula only exports galaxy, not Translator specifically.

Comment (haxxmaxx) [reply src/types.ts:2]: I see this is the way pivot does it as well

Comment (LiKang6688) [reply src/types.ts:2]: good

**Pull request merged**